fscollections/tests.py

```diff
@@ -0,0 +1,244 @@
+from django.test import TestCase
+from django.contrib.auth.models import User
+from django.urls import reverse
+
+from utils.test_helpers import create_user_and_sounds
+from fscollections.models import *
+from fscollections.forms import *
+
+# Create your tests here.
+class CollectionTest(TestCase):
+    
+    fixtures = ['licenses', 'sounds']
+
+    def setUp(self):
+        self.user = User.objects.create_user(username='testuser', email='testuser@freesound.org')
+        self.maintainer = User.objects.create_user(username='maintaineruser', email='maintainer@freesound.org')
+        self.external_user = User.objects.create_user(username='external_user', email='externaluser@freesound.org')
+        ___, ___, sounds = create_user_and_sounds(num_sounds=3, user=self.user, processing_state="OK", moderation_state="OK")
+        self.sound = sounds[0]
+        self.sound1 = sounds[1]
+        self.sound2 = sounds[2]
+        self.collection = Collection(user=self.user, name='testcollection')
+        self.collection.save()
+
+
+    def test_collections_create_and_delete(self):        
+
+        # User not logged in - redirects to login page
+        # For this to be truly useful, collection urls should be moved into accounts
+        # This might have consequences for the way collections work
+        resp = self.client.get(reverse('your-collections'))
+        self.assertEqual(resp.status_code, 302)
+
+        # Force login and test collections page
+        self.client.force_login(self.user)
+        resp = self.client.get(reverse('your-collections'))
+        self.assertEqual(resp.status_code, 200)
+
+        # User logged in, check given collection id works
+        resp = self.client.get(reverse('collection', args=[self.collection.id]))
+        self.assertEqual(resp.status_code, 200)
+
+        # Create collection view    
+        resp = self.client.post(reverse('create-collection')+'?ajax=1', {'name': 'tbdcollection'})
+        self.assertEqual(resp.status_code, 200) 
+        delete_collection = Collection.objects.get(name='tbdcollection')
+
+        # Delete collection view
```

Comment:
Are we also testing if a collection can't be deleted by a user that is not its creator? (or maintainer, whatever we decided the logic should be)